packages/shared/lib/services/proxy.service.ts

```diff
@@ -23,7 +23,7 @@ class ProxyService {
     public async routeOrConfigure(
         externalConfig: ApplicationConstructedProxyConfiguration | UserProvidedProxyConfiguration,
         internalConfig: InternalProxyConfiguration
-    ): Promise<ServiceResponse<ApplicationConstructedProxyConfiguration> | AxiosResponse | void> {
+    ): Promise<ServiceResponse<ApplicationConstructedProxyConfiguration> | AxiosResponse | AxiosError | void> {
```

Comment:
unrelated but is the function ever returning `void`?

Reply:
No it isn't, must have been carry over from earlier changes. Fixed with https://github.com/NangoHQ/nango/pull/1553/commits/91042226694d6f1671721457d462ec71c5ca6fc9